Mlem/Views/Shared/Posts/Post Sizes/Headline Post.swift

```diff
@@ -37,10 +37,23 @@ struct HeadlinePost: View {
                             StickiedTag(tagType: .community)
                         }
                         
-                        Text(post.post.name)
-                            .font(.headline)
-                            .padding(.trailing)
-                            .foregroundColor(post.read ? .secondary : .primary)
+                        VStack(alignment: .leading, spacing: AppConstants.standardSpacing) {
```

Comment:
I think it may look a little nicer if the spacing were a little tighter here. What do you think?

With `.standardSpacing:` (current):
<img src="https://github.com/mlemgroup/mlem/assets/78750526/7848c01e-9a6b-4ba3-9866-31f9ec14299c" width=50%>

With `.halfSpacing:`:
<img src="https://github.com/mlemgroup/mlem/assets/78750526/48256908-a090-429d-b859-e3222947ef70" width=50%>

Similarly for the compact post:

With `.compactSpacing`:

<img src="https://github.com/mlemgroup/mlem/assets/78750526/fa9edd90-c84a-4eff-82e2-c25c3cada7fd" width=50%>

With a spacing of 3 (half of `.compactSpacing`):

<img src="https://github.com/mlemgroup/mlem/assets/78750526/5d90c39d-f78f-47f0-a6cf-07a3d59bd53b" width=50%>

Reply:
Agreed 👍